src/components/Experiment/QuickOutput/MultiInputPreview.js

```diff
@@ -8,86 +8,99 @@ import "./MultiInput.scss";
 const defaultProps = {
   className: "multi-input-preview",
   inputs: [],
-  onBackClicked: () => {},
+  onBackClicked: () => { },
 };
 
+const inputTypes = {
+  image: "Image",
+  audio: "Audio",
+  text: "Text",
+};
 export default function MultiInputPreview(givenProps) {
   const props = { ...defaultProps, ...givenProps };
   const { getBlock, getElement } = useBEMNaming(props.className);
-
+  const getInput = () => {
```

Comment:
You added `getInput` to handle situations where there's only one input?

Reply:
I was trying to merge both inputpreview and MultiInputPreview into one and use only MultiInputPreview. Getinput takes gets input from props it seems and apparently input from props is not all CAPS. For Multiinput I am trying to take inputs from the json sent back by API( trail) and that is all caps. I will make them more organized and uniform.

Reply:
It'll be great to merge them; that will definitely be a larger undertaking that will cause merge conflicts though so you may want to do it in a separate PR.